web-animations-1/Overview.bs

```diff
@@ -3154,8 +3165,88 @@ Repeating {#repeating}
 
 </div>
 
-Core animation effect calculations {#core-animation-effect-calculations}
-----------------------------------
+#### Calculating the active duration #### {#calculating-the-active-duration}
+
+  The [=active duration=] is calculated as follows:
+
+  <blockquote>
+    [=active duration=] =
+       <code><a>iteration duration</a> × <a>iteration count</a></code>
+  </blockquote>
+
+  If either the [=iteration duration=] or [=iteration count=] are zero,
+  the [=active duration=] is zero.
+
+  Note: This clarification is needed
+  since the result of infinity multiplied by zero
+  is undefined according to IEEE 754-2008.
+
+### Direction control ### {#direction-control}
+
+  [=Animation effects=] can also be configured
+  to run iterations in alternative directions
+  using direction control.
+  For this purpose, [=animation effects=] have
+  a <dfn>playback direction</dfn> parameter
+  which takes one of the following values:
+
+  <dl dfn-for="playback direction">
+
+  : <dfn>normal</dfn>
+  ::
+      All iterations are played as specified.
+
+  : <dfn>reverse</dfn>
+  ::
+      All iterations are played in the reverse direction
+      from the way they are specified.
+
+  : <dfn>alternate</dfn>
+  ::
+      Even iterations are played as specified;
+      odd iterations are played in the reverse direction
+      from the way they are specified.
+
+  : <dfn>alternate-reverse</dfn>
+  ::
+      Even iterations are played in the reverse direction
+      from the way they are specified;
+      odd iterations are played as specified.
+
+  </dl>
+
+  The semantics of these values are incorporated into
+  the calculation of the [=directed progress=]
+  in [[#calculating-the-directed-progress]].
+
+### Easing (effect timing transformations) ### {#time-transformations}
+
+  It is often desirable to control the rate
+  at which an [=animation effect=] progresses.
+  For example, easing the rate of animation
+  can create a sense of momentum and produce a more natural effect.
+  The <a href="https://www.w3.org/TR/css-easing/">CSS Easing Functions Module</a>
+  defines [=timing functions=] for this purpose.
+  [[!CSS-EASING-1]]
+
+  [=Animation effects=] each have
+  an <dfn for="animation effect">effect timing function</dfn>,
+  which is a [=timing function=]
+  used to transform progress across the entirety of each iteration.
+  See [[#calculating-the-transformed-progress]].
+
+  The default [=effect timing function=]
+  is the [=linear timing function=].
+
+  Note: Specific [[#types-of-animation-effects|types of animation effects]]
+  may provide additional time transformations.
+  For example, [=keyframe effects=] can provide
+  [=keyframe-specific timing functions=]
+  that can apply a [=timing function=]
+  specifically between two adjacent keyframes.
+
+Calculating Progress {#core-animation-effect-calculations}
+--------------------
 
 ### Overview ### {#animation-effect-calculations-overview}
```

Comment:
I think the caption on the big diagram here needs to be updated.

It currently has:

> The first step, calculating the local time is described in § 4.6.3 Local time. Steps 2 to 4 in the diagram are described in the following sections. Steps 5 and 6 are described in § 4.7.6 Calculating the directed progress and § 4.7.7 Calculating the transformed progress respectively.

But I believe that steps 2 to 6 are now all covered in the following sections.